test-common/src/main/java/org/triplea/test/common/Assertions.java

```diff
@@ -36,11 +36,6 @@ public static void assertNotThrows(final Executable executable) {
   }
 
   private static String getCanonicalName(final Class<?> type) {
-    try {
-      final @Nullable String canonicalName = type.getCanonicalName();
-      return canonicalName != null ? canonicalName : type.getName();
-    } catch (final Throwable t) {
```

Comment:
I don't think this exception handler is required.  IIRC, I simply copied this method from JUnit when I added `assertNotThrows()`.  From looking at the code, I can't see how `Class#getCanonicalName()` could throw an `Error` or an unchecked exception that would require this exception handler.